### PR TITLE
Feature#11 setup omniauth gem and its styling

### PR DESCRIPTION
**Pull request by HikingCarrot7**
# Description
What did you implemented/Why did you implemented this:
 - Implemented Log in / Sign up with Google.
 - Implemented Log in / Sign up with Github.
 
  ## Screenshots
login

signup

### CODE REVIEW

**Review by AgustinPalmaM — APPROVED** [review]
lgtm

File: config/credentials.yml.enc [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+nQ/eeGMlCwW/iU20o5aDEXXRygvr45shC0uYTmw7d9vsUgywQ57NfyhQCz8/0nKd5p10zLPOEvHCUxpBAD4b7OnXUzu/55br8WEhJ/CvGI7pOIVm4WliGaZDbaqozDi5NeTED5etRtwAt2HlcPoGQJ0GYLrCntyEwSzuqR0NVAxxx0O7FRbv0Jh6q/s1CBRTUX5rQdcUaiyogNbtPwq/aXiwWFIKbP5jf4aCRbT+rjxW/FVdlNrr9jXOooW5ca+Fykz8TekzwtFqwBwUuUJRVyFLxQwXRV5vY5JN8LYhCpO730jldDKnp1kGOsYBS/V5I8cHP+DCY4wo9FHFMwBRZibOCwRHFcgmQGKGOUxJXoajXyZs4z1TcNc8v4eSgwh/uV5vsiGtxaXKpMmGF0etr+O/tUOI2K13D/aXCyyjthY95RNgMNhZWiO528+VJFjyJXlR5fei1YEy1CcL1CiBjZCRLVa3vkp7ccFjpIVTUPI/ERvmGe1bmO/0r95jHXj6rlr19K69G+268GP+7tA7lBiNbXha5qFog9fRGcHcoNN1BQnZDIklbZuXZ3zsZcejzlHQwti1vnw/vzFM1rrSKSXf/+0jJdQj4N9zh7PJ2cuYC8SP3IdHKBYYoz/QoguDhSzRVsk2o/Nf0l3ob8Un3AQ46IHV3yqc1RAvm9x2OKtAZOnDAtpwQsIKbvLdgcQVj80miczAWlNHBx5Hb/XDtXp1FVc7mAcGEWp35rZXIKLC7Lbr8xqAliYNzoZMUhq9JNc8a5t7KinqaQ==--wtK5kfuiXmniMX4f--G+YJLHXgjiyXJFBSIzHSfA==
```
Comment (V-Juarez) [inline config/credentials.yml.enc:1]: Add new line at the end

**Review by V-Juarez — APPROVED** [review]
Nice work

File: config/credentials.yml.enc (hunk 1 above)
Comment (guillermo-delprado) [reply config/credentials.yml.enc:1]: 👍

**Pull request merged**